platform-designs/design-content/copy-AutSPACEs-website.md

```diff
@@ -0,0 +1,129 @@
+# Copy for AutSPACEs website
+Status: WIP
+
+* [Content for “About Autism” sensory processing, and what the project is](#Content-for-“About-Autism”-sensory-processing,-and-what-the-project-is) 
+* [Content for “How to Use the Platform”](#Copy-for-'How-to-use-the-platform')
+* [Content for “Open Humans” explanation for redirection](#Content-for-“Open-Humans”-explanation-for-redirection) 
+* [Content for “How we use your data’](#How-we-use-your-data)
+* [Content for “Get Involved” - explanation of participatory science and links to take part as researchers](#“Get-Involved”)
+
+---
+
+# Content for “About Autism” sensory processing, and what the project is 
+
+
+## What is autism?
+
+Autism affects the way people communicate and experience the world around them.
+
+Autism is a spectrum of developmental conditions, including Asperger’s Syndrome. Every autistic person is different. Some are able to learn, live and work independently, while many have learning differences or co-occurring health conditions that require specialist support.
```

Comment:
Aspberger's syndrome has been folded into ASD in newer DSM versions

Reply:
On review - I'm wondering if this piece is something we can give a refresh in parts - more reflective of a social model than a medical model - it'll be a tricky one but an important one.

Reply:
For now I've taken out the `including Asperger’s Syndrome.` from line 19.